acceptance/openstack/client_test.go

```diff
@@ -33,6 +33,9 @@ func TestAuthenticatedClient(t *testing.T) {
 	if cc.DomainID == "" {
 		t.Errorf("Domain ID is not set for the client")
 	}
+	if cc.RegionID == "" {
```

Comment:
`RegionName` produces issue with `cc` CloudAndClient Interface, because it has `RegionName`. `openstack.Cloud` has `RegionName`
```go
// cc stands for `cloud` & `client`
type cc struct {
	*openstack.Cloud
	*golangsdk.ProviderClient
}
```